code/modules/surgery/bodyparts/species_parts/lizard_bodyparts.dm

```diff
@@ -8,28 +8,42 @@
 	limb_id = SPECIES_LIZARD
 	is_dimorphic = TRUE
 
-/obj/item/bodypart/l_arm/lizard
+/obj/item/bodypart/arm/l_arm/lizard
 	icon_greyscale = 'icons/mob/species/lizard/bodyparts.dmi'
 	limb_id = SPECIES_LIZARD
+	unarmed_attack_verb = "slash"
+	unarmed_attack_effect = ATTACK_EFFECT_CLAW
+	unarmed_attack_sound = 'sound/weapons/slash.ogg'
+	unarmed_miss_sound = 'sound/weapons/slashmiss.ogg'
 
-/obj/item/bodypart/r_arm/lizard
+/obj/item/bodypart/arm/r_arm/lizard
 	icon_greyscale = 'icons/mob/species/lizard/bodyparts.dmi'
 	limb_id = SPECIES_LIZARD
+	unarmed_attack_verb = "slash"
+	unarmed_attack_effect = ATTACK_EFFECT_CLAW
+	unarmed_attack_sound = 'sound/weapons/slash.ogg'
+	unarmed_miss_sound = 'sound/weapons/slashmiss.ogg'
 
-/obj/item/bodypart/l_leg/lizard
+/obj/item/bodypart/arm/l_arm/lizard/ashwalker
+	chunky = TRUE
+
+/obj/item/bodypart/arm/r_arm/lizard/ashwalker
+	chunky = TRUE
```

Comment:
I really do not want to encourage this kind of behavior.

Reply:
What kind of behavior?